library/src/rtc_stockham_gen.cpp

```diff
@@ -297,10 +314,21 @@ std::string stockham_rtc(const StockhamGeneratorSpecs& specs,
     {
         std::unique_ptr<StockhamKernel> kernel;
         if(scheme == CS_KERNEL_STOCKHAM)
-            kernel = std::make_unique<StockhamKernelRR>(specs);
+        {
+            if(ppType == PartialPassType::PPT_SBRR)
```

Comment:
Aren't there three cases here?  A switch statement would defend against missed options.

Reply:
Thanks, I've reorganized the code and now the three cases are addressed.